.github/workflows/ci.yml

```diff
@@ -44,22 +58,43 @@ jobs:
           large-packages: true
           docker-images: true
           swap-storage: true
-      - name: Install lld & GDAL & Protobuf
-        run: |
-          sudo apt-get update
-          sudo apt-get install lld libgdal-dev gdal-bin build-essential clang curl protobuf-compiler
-          sudo apt-get clean
-          export C_INCLUDE_PATH=/usr/include/gdal:$C_INCLUDE_PATH
-          export CPLUS_INCLUDE_PATH=/usr/include/gdal:$CPLUS_INCLUDE_PATH
-          sudo ldconfig
-      - name: Install Rustup
+      # TODO: Get Cache back
```

Comment:
create a card for this(?)

Reply:
https://trello.com/c/Cm8NNqkX/1810-optimize-backend-ci-mit-cache-und-tests